contact.html

```diff
@@ -0,0 +1,121 @@
+<!doctype html>
```

Comment:
Good job including the contact page, it looks like you didn't get around to changing things around but great to see that you set it up!